docs/Using Fleet/GitOps.md

```diff
@@ -569,16 +569,44 @@ Can only be configured for all teams (`org_settings`).
 
 #### mdm
 
-The `mdm` section lets you enable MDM features in Fleet.
+##### apple_business_manager
 
-- `apple_bm_default_team` - is name of the team that macOS hosts in Apple Business Manager automatically enroll to when they're first set up. If empty, hosts will enroll to "No team" (default: `""`).
+- `organization_name` - is the organziation name associated with the Apple Business Manager account.
+- `macos_team` - macOS hosts are automatically added to this team in Fleet when they appear in Apple Business Manager.
+- `ios_team` - iOS hosts are automatically added to this team in Fleet when they appear in Apple Business Manager.
+- `ipados_team` - iPadOS hosts are automatically added to this team in Fleet when they appear in Apple Business Manager.
 
 ##### Example
 
 ```yaml
 org_settings:
   mdm:
-    apple_bm_default_team: "Workstations" # Available in Fleet Premium
+    apple_business_manager: # Available in Fleet Premium
+    - organization_name: Fleet Device Management Inc.
+      macos_team: "💻 Workstations" 
+      ios_team: "📱🏢 Company-owned iPhones"
+      ipados_team: "🔳🏢 Company-owned iPads"
+```
+
+Can only be configured for all teams (`org_settings`).
+
+##### volume_purchasing_program
+
+- `location` - is the name of the location in the Apple Business Manager account.
+- `teams` - is a list of team names. If you choose specific teams, App Store apps in this VPP account will only be available to install on hosts in these teams. If not specified, App Store apps are available to install on hosts in all teams.
+
+##### Example
+
+```yaml
+org_settings:
+  mdm:
+    volume_purchasing_program: # Available in Fleet Premium
+    - location: Fleet Device Management Inc.
+      teams: 
```

Comment:
### Dev note

If the user tries to add a team that doesn't exist, show the following error: 

```
Couldn't edit org_settings.mdm.volume_purchasing_program. "💻 Workstations" team doesn't exist.
```

Reply:
QA note: This will likely happen when a user changes a team's name via GitOps but forgets to update the team name here.

Reply:
### Dev note

If the user tries to add a team that already has a VPP token, show the following error:

```
Couldn't edit org_settings.mdm.volume_purchasing_program. "💻 Workstations" team already has a VPP token.  Each team can only have on VPP token.
```

Reply:
Hey @roperzh, now that a new VPP token doesn't have any teams by default (instead of "All team") how do you think the user should specify "All teams" via GitOps?

Could we make it so empty `teams` for a VPP token means it's available for "All teams." And once, the user adds a second VPP token we error and say you have to assign specific teams.

Reply:
Hey @noahtalerman that makes sense, the only gotcha I see is how would you define "no team"?

Reply:
I think they would specify `"No team"` which is a special, reserved team name in Fleet.

That made me wonder, what happens if I create a team w/ "No team" name in Fleet...

It actually breaks the team. And so does naming a team "All teams": https://github.com/fleetdm/fleet/issues/21264

Reply:
@roperzh I think we want to add the reserved "All teams" and "No teams" teams as part of this story. 

This way, the IT admin can specify these here in GitOps.

I think let's track this effort as part of the bug: #21264

What do you think?